### PR TITLE
Remove unused eject from SyncControl

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

File: src/track/beatfactory.cpp [hunk 1]
```diff
@@ -95,6 +95,14 @@ mixxx::BeatsPointer BeatFactory::makePreferredBeats(
                 sampleRate, subVersion, constBPM, firstBeat * 2);
         return pGrid;
     } else if (version == BEAT_MAP_VERSION) {
+        if (beats.size() > 48) {
```
Comment (Holzhaus) [inline src/track/beatfactory.cpp:98]: I think this unrelated and slipped in by mistake.

Comment (daschuer) [reply src/track/beatfactory.cpp:98]: Oh yes.

Comment (daschuer) [reply src/track/beatfactory.cpp]: Forced pushed without that change.

**Review by Holzhaus — APPROVED** [review]
LGTM, thank you. Waiting for CI.

**Pull request merged**